.github/workflows/contracts.yml

```diff
@@ -110,6 +118,14 @@ jobs:
       - name: Install dependencies
         run: yarn install --frozen-lockfile
 
+      # We need this step because the `@keep-network/tbtc` which we update in
```

Comment:
NB and not this PR but a general comment to that workflow. We are probably referring to the wrong package because `@keep-network/tbtc` is actually v1 which is not used from within v2. We should probably produce a new package called `@keep-network/tbtc-v2`